hoot-core/src/main/cpp/hoot/core/algorithms/alpha-shape/AlphaShapeGenerator.cpp

```diff
@@ -61,6 +61,7 @@ OsmMapPtr AlphaShapeGenerator::generateMap(OsmMapPtr inputMap)
   OsmMapPtr result;
 
   result.reset(new OsmMap(inputMap->getProjection()));
+  result->appendSource(inputMap->getSource());
```

Comment:
Sources need to be transferred from the `inputMap` to the `result` map object so that the alpha-shape output contains the source for the `input_src` attribute.